src/main/scala/beam/agentsim/agents/vehicles/BeamVehicle.scala

```diff
@@ -78,8 +66,8 @@ object BeamVehicle {
       0.0
   }
 
-  def buildActorName(vehicleId: Id[Vehicle]): String = {
-    s"$ActorPrefixName${vehicleId.toString}"
+  def buildActorName(matsimVehicle: Vehicle): String = {
+    s"$ActorPrefixName${matsimVehicle.getType.getDescription}-${matsimVehicle.getId.toString}"
```

Comment:
We can use the "description" field however we want, so I'm planning to use it like a name or code field (e.g. "CAR", "BIKE").